crates/vm/levm/src/vm.rs

```diff
@@ -880,6 +880,15 @@ impl VM {
 
         self.prepare_execution(&mut initial_call_frame)?;
 
+        // If the transaction is a CREATE transaction and the address is already in the database and is not empty
+        // transaction should be reverted.
+        if self.is_create() {
+            let new_address_acc = self.db.get_account_info(initial_call_frame.to);
+            if new_address_acc.has_code() || new_address_acc.has_nonce() {
```

Comment:
It is an `||`. We created the function because of the [specs](https://ethereum.github.io/execution-specs/src/ethereum/cancun/state.py.html#ethereum.cancun.state.account_has_code_or_nonce:0). It may be a little bit overkill but we can use it I guess.

Reply:
The type of `new_address_acc` is `AccountInfo`. This type doesn't have that method :( Maybe we can implement it in the future.

Reply:
> It is an `||`. We created the function because of the [specs](https://ethereum.github.io/execution-specs/src/ethereum/cancun/state.py.html#ethereum.cancun.state.account_has_code_or_nonce:0). It may be a little bit overkill but we can use it I guess.

Noted, thank you for the link Jere! 

> The type of `new_address_acc` is `AccountInfo`. This type doesn't have that method :( Maybe we can implement it in the future.

Oh, my mistake Dami.